pyaedt/hfss.py

```diff
@@ -4561,6 +4565,49 @@ def assign_hybrid_region(self, obj_names, boundary_name="", hybrid_region="SBR+"
             bound.props["Type"] = hybrid_region
         return bound
 
+    @pyaedt_function_handler()
+    def assign_febi(self, obj_names, boundary_name=""):
+        """Assign an FE-BI region to one or more objects.
+
+        Parameters
+        ----------
+        obj_names : str or list or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+            One or more object names or IDs.
+        boundary_name : str, optional
+            Name of the boundary. The default is ``""``.
```

Comment:
Perhaps we should include the bevavior that occurs when ``""`` is used, something like this:
The default is ``""``, in which case a name is automatically assigned.